internal/worker/worker.go

```
@@ -0,0 +1,75 @@
+}
+
+func CreateWorker() (*Worker, error) {
+	logger.Info(common.COMPONENT_WORKER, "Creating Worker...")
+
+	worker := &Worker{
+		getDefaultWorkerConfig(),
+	}
+
+	ticker := time.NewTicker(time.Second * time.Duration(worker.HeartbeatInterval))
+	go func() {
+		for {
```

golangci-lint (1.64.8)

S1000: should use for range instead of for { select {} }

(gosimple)